How-to-access-Oppia-webpages.md

```diff
@@ -1,237 +1,336 @@
-# How to access Oppia webpages
+## Table of Contents
+
+* [Log in](#log-in)
+* [Log in as a super-administrator](#log-in-as-a-super-administrator)
+  * [Using an email with implicit super-admin privileges](#using-an-email-with-implicit-super-admin-privileges)
+  * [Grant super-admin privileges after account creation](#grant-super-admin-privileges-after-account-creation)
+* [Access pages](#access-pages)
+  * [Admin page](#admin-page)
+    * [Assign roles](#assign-roles)
+  * [About page](#about-page)
+  * [Collection editor page](#collection-editor-page)
+  * [Collection player page](#collection-player-page)
+  * [Contact Page](#contact-page)
+  * [Creator dashboard page](#creator-dashboard-page)
+  * [Contributor dashboard page](#contributor-dashboard-page)
+  * [Delete account page](#delete-account-page)
+  * [Donate page](#donate-page)
+  * [Email dashboard page](#email-dashboard-page)
+  * [Exploration editor page](#exploration-editor-page)
+  * [Exploration player page](#exploration-player-page)
+  * [Get started page](#get-started-page)
+  * [Landing pages](#landing-pages)
+  * [Skill editor page](#skill-editor-page)
+  * [Story editor page](#story-editor-page)
+  * [Story player page](#story-player-page)
+  * [Topic editor (including preview tab)](#topic-editor-including-preview-tab)
+  * [Preferences page](#preferences-page)
+  * [Profile page](#profile-page)
+  * [Community library page](#community-library-page)
+  * [Topics and skills dashboard page](#topics-and-skills-dashboard-page)
+  * [Thanks page](#thanks-page)
+  * [Terms page](#terms-page)
+
 Oppia has many webpages and this is a comprehensive guide on how to access all those pages. Before a contributor makes a PR, we expect that the contributor has thoroughly tested the changes made in the PR for functional correctness. Part of this process is manually testing any pages that are affected by their code.
 
-## Sign Up (As Administrator)
+## Log in
 
 Many Oppia Pages require authentication to access. In order to sign in:
-1. Click the sign-in button on the top left navigation bar
-![Sign-In Button](images/Webpage-Guide/signInButton.png)
 
-2. Once redirected to the login page, click the **Sign in as Administrator checkbox** and then login.
+1. Click the sign-in button on the top left navigation bar.
+
+   ![Sign-In Button](images/Webpage-Guide/signInButton.png)
+
+2. Once redirected to the login page, choose an email address and sign in. If you have used this email address before, you will be signed in to the existing account. Otherwise, a new account will be created for you.
+
+   ![Sign-in page](images/Webpage-Guide/nonAdminSignUp.png)
+
+## Log in as a super-administrator
+
+There are two kinds of administrators in Oppia: super-administrators have access to the admin page, where they can set the role of any user. One of these roles is the administrator role, which gives the user access to all other privileged pages (like the topics and skills dashboard).
+
```

Comment:
We are removing the `administrator role` in PR https://github.com/oppia/oppia/pull/12989 [About to merge] After these changes we will just have super_admin. Note: we also have multiple sub-admin roles: Translation admin, Voiceover admin, Questions admin etc.

Reply:
The PR has failing backend tests, so this PR might be ready to merge first. How about we merge this PR and then update the docs once oppia/oppia#12989 merges? If oppia/oppia#12989 merges first, then we can update this PR before merging

Reply:
The PR got merged yesterday! Let's update this page :)

Reply:
done